theme/stencil/src/components/copilot-sidebar/copilot-sidebar.tsx

```diff
@@ -0,0 +1,35 @@
+import { Component, h, Prop, State, Host } from '@stencil/core';
+import {gb} from "../../util/util";
+
+@Component({
+  tag: 'copilot-sidebar',
+  styleUrl: 'copilot-sidebar.css',
+  shadow: false,
+})
+export class CopilotSidebar {
+  // The copilotSrc is URL where the copilot sidebar is hosted
+  @Prop()
+  copilotSrc: string;
+
+  @State()
+  showSidebar = false;
+
+  
+  componentWillLoad() {
+    this.showSidebar = gb.isOn('copilot-on-docs');
```

Comment:
Can you explain who sets this GB flag? ('copilot-on-docs')

Reply:
This is a feature flag i set up on GrowthBook:
<img width="1263" alt="image" src="https://github.com/user-attachments/assets/bae34a01-9db7-421e-a69f-edf9c067721c">

We can then link the feature flag to an experiment:
<img width="1202" alt="image" src="https://github.com/user-attachments/assets/fa2054dc-61f1-4ee7-bcfe-068e4fbfbfb0">